lib/operations/command.rb

```diff
@@ -143,8 +141,11 @@ class OperationFailed < StandardError
   option :form_base, Operations::Types::Class, default: proc { ::Operations::Form }
   option :form_class, Operations::Types::Class, default: proc { build_form }
   option :form_hydrator, Operations::Types.Interface(:call), default: proc { FORM_HYDRATOR }
-  option :error_reporter, Operations::Types.Interface(:call), default: proc { ERROR_REPORTER }
-  option :transaction, Operations::Types.Interface(:call), default: proc { TRANSACTION }
+  option :info_reporter, Operations::Types::Nil | Operations::Types.Interface(:call),
```

Comment:
info_reporter or maybe just `logger` or `info_logger` ? 🤔

Reply:
I was thinking about it but we have `error_reporter` already so I followed the pattern.

Reply:
I actually have an idea of introducing just `logger` option later which will internally decided where to log based on, say, level. But maybe not :)